server/i18n/en/errors.json

```diff
@@ -160,6 +160,10 @@
     }
   },
   "bedspaceSearch": {
+    "probationDeliveryUnits": {
+      "empty": "You must choose at least one PDU",
+      "maxNumberProbationDeliveryUnits": "Select 1 to 5 PDUs"
```

Comment:
Where is the functionality to return `maxNumberProbationDeliveryUnits` when more than 5 are selected?

Reply:
I presumed this validation was done in the FE - but I can see it's done in the API.  Is this correct?

Reply:
@danhumphreys-moj the validation is done in the API and this file is used to translate the error code to a human readable format. We agreed that all validations should really be done in the API instead of the frontend because if there is a different client using the API we would have to duplicate all validations. Its better do do that on the backend.